src/transformState.js

```diff
@@ -49,7 +49,23 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
-}
+  transforms.forEach(function(item, i, arr) {
```

Comment:
why you need `arr` and `i`?
use arrow function for method
```suggestion
  transforms.forEach((item) => {
```